apps/snp_store/snp_store.cpp

```diff
@@ -81,39 +96,42 @@ bool loadGenomes(TGenomeSet &genomes, StringSet<CharString> &fileNameList, ::std
         Dna5String seq;
         for (; !atEnd(seqFileIn); ++seqCount)
         {
-            readRecord(temp, seq, seqFileIn);
+            try
+            {
+                readRecord(temp, seq, seqFileIn);
+            }
+            catch (ParseError const & e)
+            {
+                //std::cout << e.what() << "Base will be considered as 'N'." << std::endl;
```

Comment:
If I remember correctly, the standard output was flooded with error messages I used SNPStore on a FASTA file containing not only A,C,G,T,N.  I fixed it now, s.t. the warning will only be written to ouput when it occurs for the first time.